.docker/app/Dockerfile

```diff
@@ -43,11 +43,11 @@ RUN apk add -U --no-cache \
     if [ "${RAILS_ENV}" = "production" ]; then rm -rf tmp/*; fi && \
     if [ "${RAILS_ENV}" = "production" ]; then apk del build-dependencies; fi && \
     rm -rf /var/cache/apk/ && \
-    rm -rf /usr/local/share/.cache/yarn && \
-    chown nobody -R /app/tmp && \
-    chown nobody -R /app/log
+    rm -rf /usr/local/share/.cache/yarn  && \
+    mkdir -p /secure-tmp && chmod 700 /secure-tmp && \
```

Comment:
I think this should be added to entry-point script because volume mounts are not availabe until after the image is being used in a container.